### ISSUE TITLE
Show bitwsap stats / wantlist

### ISSUE THREAD

**Issue by walkah** [fission-suite/benchmark#1]
See: https://talk.fission.codes/t/debugging-webnative-traffic/1440

This should be built-in to show stats and wantlist for debugging.

**Comment by icidasset** [comment]
If we want to show it in the app itself, we'll need some additional infrastructure (setup additional `postMessage` calls).
Or make more PRs for https://www.npmjs.com/package/ipfs-message-port-client and the associated libraries.

The reason why this is needed is because this info needs to come from the `js-ipfs` instance, which is only available in the (shared) web worker. That's why you have to access it like explained in the post. In the webnative apps we use a proxy (a fork of the client library linked above), and that proxy has no way to request that information from the js-ipfs instance.

**Comment by walkah** [comment]
Yeah, I was just reading through ipfs-message-port-client! Thanks for confirming 🙏
